packages/cli/src/metadataGeneration/typeResolver.ts

```diff
@@ -695,6 +695,11 @@ export class TypeResolver {
     } else {
       const declarations = this.getModelTypeDeclarations(type);
 
+      if (declarations.length === 0) {
+        // Escape some types like Date.
+        return name;
+      }
```

Comment:
If we dont find them, we should just not support it. This may work by a lucky accident, but is probably a huge footgun.

Reply:
Maybe we can just escape for `Date` ?